src/main/java/seedu/address/logic/parser/ImportCommandParser.java

```diff
@@ -69,9 +70,9 @@ public ImportCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_FOLDER_SPECIFIED, file.getPath()));
         }
 
-        List<Person> toAdd = new ArrayList<>();
+        List<Person> personToAddList = new ArrayList<>();
 
-        List<String> invalidFields = new ArrayList<>();
+        List<String> invalidFieldsList = new ArrayList<>();
```

Comment:
why need to add the word list?

Reply:
Was trying to match with personToAddList but i think the original is fine. Will revert back